aspnet/tutorials/nano-server.rst

```diff
@@ -1,8 +1,164 @@
-.. include:: /../common/stub-topic.txt
+.. _nano-server:
 
-|stub-icon| ASP.NET 5 on Nano Server
-====================================
+ASP.NET 5 on Nano Server
+========================
+
+By `Sourabh Shirhatti`
+
+.. attention:: This tutorial uses a pre-lease version of the Nano Server installation option of Windows Server Technical Preview 4. You may use the software in the virtual hard disk image only to internally demonstrate and evaluate it. You may not use the software in a live operating environment. Please see https://go.microsoft.com/fwlink/?LinkId=624232 for specific information about the end date for the preview.
+
+In this tutorial, you'll take an existing ASP.NET 5 application and deploy it to a Nano Server instance running IIS.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Introduction
+------------
+
+Windows Server 2016 Technical Preview offers a new installation option: Nano Server. Nano Server is a remotely administered server operating system optimized for private clouds and datacenters. It takes up far less disk space, sets up significantly faster, and requires far fewer updates and restarts than Windows Server. You can learn more about Nano Server from the `official docs <https://msdn.microsoft.com/en-us/library/mt126167.aspx>`_.
+
+In this tutorial, we will be using the pre-built `Virtual Hard Disk (VHD) for Nano Server <https://msdn.microsoft.com/en-us/virtualization/windowscontainers/nano_eula>`_  from Windows Server Technical Preview 4. This pre-built VHD already includes the Reverse Forwarders and IIS packages which are required for this tutorial.
+
+Before proceeding with this tutorial, you will need the published output of an existing ASP.NET 5 application. Refer to the guidance on :ref:`publishing <dnu-publish>`. Ensure your application is built targeting the **64-bit** version of ``coreclr``.
```

Comment:
> Before proceeding with this tutorial, you will need the published output of an existing ASP.NET 5 application. Refer to the guidance on :ref:`publishing <dnu-publish>`. Ensure your application is built targeting the **64-bit** version of ``coreclr``.

Before proceeding with this tutorial, you will need the :ref:`published <dnu-publish>` output of an existing ASP.NET 5 application. Ensure your application is published with the **64-bit** version of ``coreclr``.